LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2Ability_LW_ChosenAbilities.uc

```diff
@@ -1355,17 +1397,20 @@ static function X2DataTemplate CreateChosenKidnap()
 	Template.bShowActivation = true;
 
 	ImmuneUnitCondition = new class'X2Condition_UnitType';
-	ImmuneUnitCondition.ExcludeTypes.AddItem('RebelSoldierProxy');
-	ImmuneUnitCondition.ExcludeTypes.AddItem('RebelSoldierProxyM2');
-	ImmuneUnitCondition.ExcludeTypes.AddItem('RebelSoldierProxyM3');
+	
+	foreach default.KIDNAP_ELIGIBLE_SOLDIERS(SoldierTemplateName)
+	{
+		ImmuneUnitCondition.IncludeTypes.AddItem(SoldierTemplateName);
```

Comment:
I suggest changing the name of `ImmuneUnitCondition` because the included types are *not* immune, and so the variable is confusingly named.